Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM python:2.7-alpine3.6
+
+ENV DJANGO_SETTINGS_MODULE=core.settings_dev
+
+RUN apk update && \
+    apk -U upgrade && \
+    apk add \
+        gcc \
+        musl-dev \
+        jpeg-dev \
+        gcc-avr \
+        g++ \
+        bash \
+        libxml2-dev \
+        libxslt-dev \
+        readline-dev \
+        python-dev \
+        linux-headers \
+        ncurses-dev \
+        patch \
+        libffi-dev \
+        openssh-client \
+        supervisor \
+        mariadb-dev && \
+    rm -rf /var/cache/apk/* && \
+    ln -s /usr/include/locale.h /usr/include/xlocale.h && \
+    mkdir /rua && \
+    mkdir -p /usr/lib/readline && \
+    mkdir -p /run/nginx && \
+    mkdir -p /var/log/gunicorn && \
+    mkdir -p /var/log/nginx && \
+    ln -s /usr/lib/libreadline.a /usr/lib/readline/libreadline.a && \
+    ln -s /usr/lib/libhistory.a /usr/lib/readline/libhistory.a
+
+COPY . /rua
```

Comment:
As mentioned in another comment below, this should be `COPY ./src /app`. Since we are not copying the requirements in this case, we will have to add another instruction to copy `requirements.txt` before installing them.